packages/vue-client/src/components/GameCreation/GameCreationForm.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import { ref, type Ref, watch } from "vue";
+import {
+  getVariantList,
+  getDefaultConfig,
+type BadukConfig,
+type ParallelGoConfig,
+} from "@ogfcommunity/variants-shared";
+import * as requests from "@/requests";
+import router from "@/router";
+import BadukConfigFormVue from "./BadukConfigForm.vue";
+import type { BadukWithAbstractBoardConfig } from "@ogfcommunity/variants-shared/src/badukWithAbstractBoard/badukWithAbstractBoard";
+import BadukWithAbstractBoardConfigFormVue from "./BadukWithAbstractBoardConfigForm.vue";
+import  ParallelGoConfigFormVue from "./ParallelGoConfigForm.vue";
+
+const variants: string[] = getVariantList();
+const variant: Ref<string> = ref(variants.length ? variants[0] : "");
+var config: object;
+const configString: Ref<string> = ref("");
+watch(
+  variant,
+  () => {
+    config = getDefaultConfig(variant.value);
+    configString.value = JSON.stringify(config, null, 2);
+  },
+  { immediate: true }
+);
+
+const createGame = async () => {
+  console.log('config:');
+  
+  console.log(config);
+  
+  const game = await requests.post("/games", {
+    variant: variant.value,
+    config: config,
+  });
+  console.log(game);
+  router.push({ name: "game", params: { gameId: game.id } });
+};
+
+const parseConfigThenCreateGame = async () => {
+  config = JSON.parse(configString.value);
+  await createGame();
+}
+
+const setConfig = (newConfig: object) => config = newConfig;
+</script>
+
+<template>
+  <div className="game-creation-form">
+    <h3>Create a game</h3>
+    <div>
+      <label>Variant:</label>
+      <select v-model="variant">
+        <option v-for="variant in variants" :key="variant">
+          {{ variant }}
+        </option>
+      </select>
+    </div>
+    <label>Config: </label>
+    <template v-if="variant==='baduk' || variant==='phantom' || variant=='capture'">
```

Comment:
Just a thought..

It might nice if we could have a mapping from Variants to ConfigViews, kind of like how we map variants to boards ([board_map.ts](https://github.com/benjaminpjones/govariants/blob/main/packages/vue-client/src/board_map.ts)).  The benefits would be brevity and separation of concerns.

But that is not a blocker, I think it looks pretty good as-is!

Reply:
ok I added a map from variants to the form components, named config_form_map

also ran yarn lint and removed sign restriction from komi inputs.